extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/text_generation.py

```diff
@@ -98,7 +100,7 @@ def construct_stream_output(
         output = ExecuteResult(
             **{
                 "output_type": "execute_result",
-                "data": copy.deepcopy(accumulated_message),
```

Comment:
Why did this do a deepcopy while the updated one doesn't?

Reply:
I think `accumulated_message` is a message so this isn't necessary. It gets updated from `data`, which itself is also a `str`